confdb/confdb.go

```diff
@@ -691,7 +864,11 @@ func (v *View) Unset(databag Databag, request string) error {
 
 func (v *View) matchWriteRequest(request string) ([]requestMatch, error) {
 	var matches []requestMatch
-	subkeys := strings.Split(request, ".")
+	subkeys, err := splitViewPath(request)
```

Comment:
as mentioned elsewhere some of this should use something that produce accessors? can you leave at least TODO where you think it's the case?